services/ui/ws/window_manager_display_root.cc

```diff
@@ -25,9 +25,24 @@ WindowManagerDisplayRoot::WindowManagerDisplayRoot(Display* display)
   properties[mojom::WindowManager::kName_Property] =
       std::vector<uint8_t>(name.begin(), name.end());
 
-  root_.reset(window_server()->CreateServerWindow(
-      window_server()->display_manager()->GetAndAdvanceNextRootId(),
-      properties));
+  // We pretend the WindowManagerDisplayRoot::root_ is created by the root
```

Comment:
I believe it belongs to its own fixup!, yes. the reason for leaving it separate, was for an easy revert, if we decide to.